logstage/logstage-routing/src/test/scala/com/github/pshirshov/izumi/logstage/api/LoggingMacroTest.scala

```diff
@@ -76,11 +82,16 @@ object LoggingMacroTest {
   val jsonPolicy = new JsonRenderingPolicy()
   val consoleSinkText = new ConsoleSink(coloringPolicy)
   val consoleSinkJson = new ConsoleSink(jsonPolicy)
+  val fileSinkText = new FileSink(simplePolicy, consoleSinkText, FileSinkConfig(2, "logstage", Rotation.EnabledRotation(2)))
```

Comment:
It could be useful to be able to run tests without doing file I/O using DummyFiles.